package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "firestore-service",
+  "version": "1.0.1",
+  "description": "Create firestore queries with simple HTTP methods",
+  "private": true,
+  "scripts": {
+    "test": "jest --coverage --detectOpenHandles",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Wolox/firestore-service.git"
+  },
+  "author": "Lucas Zibell <lucas.zibell@wolox.com.ar>",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Wolox/firestore-service/issues"
+  },
+  "homepage": "https://github.com/Wolox/firestore-service#readme",
+  "dependencies": {
+    "eslint": "^5.0.1",
+    "eslint-plugin-jest": "^21.17.0",
+    "firebase": "^5.2.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.26.0",
+    "babel-eslint": "^8.2.5",
+    "babel-jest": "^23.2.0",
+    "babel-loader": "^7.1.4",
+    "babel-preset-env": "^1.7.0",
+    "dotenv": "^6.0.0",
+    "eslint-config-airbnb": "^17.0.0",
+    "eslint-config-prettier": "^2.9.0",
+    "eslint-plugin-flowtype": "^2.49.3",
+    "eslint-plugin-import": "^2.13.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-prettier": "^2.6.1",
+    "eslint-plugin-react": "^7.10.0",
```

Comment:
Remember to check the dependencies, there are some of them you don't really need